pkg/volume/util/util.go

```diff
@@ -945,6 +945,38 @@ func CheckPersistentVolumeClaimModeBlock(pvc *v1.PersistentVolumeClaim) bool {
 	return utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) && pvc.Spec.VolumeMode != nil && *pvc.Spec.VolumeMode == v1.PersistentVolumeBlock
 }
 
+// IsWindowsUNCPath checks if path is prefixed with \\
+// This can be used to skip any processing of paths
+// that point to SMB shares, local named pipes and local UNC path
+func IsWindowsUNCPath(goos, path string) bool {
+	if goos != "windows" {
+		return false
+	}
+	// Check for UNC prefix \\
+	if strings.HasPrefix(path, `\\`) {
+		return true
+	}
+	return false
+}
+
+// IsWindowsLocalPath checks if path is a local path
+// prefixed with "/" or "\" like "/foo/bar" or "\foo\bar"
+func IsWindowsLocalPath(goos, path string) bool {
```

Comment:
Should `MakeAbsolutePath` call this function instead?

Reply:
@yujuhong I would prefer to keep `MakeAbsolutePath` as is for now in the context of this PR. The scope of this PR is to enable named pipe mounting support in Windows and that code path skips `MakeAbsolutePath`. `MakeAbsolutePath` is called in a couple of contexts (not involving named pipes) and maybe refactored in a separate PR.